popup-portfolio.css

```diff
@@ -0,0 +1,285 @@
+.popuppt {
```

Comment:
@1GORDON 
Hi Gordon, I have realized that I did not update the corrections when I requested the review, can you please take a look? Also:
1. Which size screen are you working with? I worked with the two size screens that the project required, not sure which one you are using.
2. Having the site responsive is not a requirement for this project, I am doing my best in order to get this approved, but not sure which screen you are working, so that I can fix the issue. 
3. The last 2 bullet points were already fixed, can you please check to see if I can request the final review?

Reply:
@1GORDON  can you please provide me with more insights please? I needed before asking a third code review.